openapi/vendor/oci-schemas/fs.go

```diff
@@ -0,0 +1,2611 @@
+SkymOhoen7CIE68yIBu2mobPRUw0bTRSQSvLZ3NV10vgyrefVRpY81Bb+lkN1YnEKsV1tWLKldI/zUN9
+VqnAraz9UpXzShCWRM7lwrY6PdvvJGOrSPti6pms+F3ZGeHLmMz4W2y83xvSoXLptPDZza2ZWRQa7VFn
+pHd0ZFLf+cbEvZkQocknToiiYxpRNc22eh55KFZuhWfcvUGhyRKz3T4jMSrBpxh5GT1q5nJDwoLVI94S
+83RWmuf00imL7BqFmIrI4luG/7WAl+jk9+ag/gycs73/BQAA//9jKv+b8DAAAA==
+`,
+	},
+
+	"/image-manifest.schema.json": {
+		name:    "image-manifest.schema.json",
+		local:   "image-manifest.schema.json",
+		size:    6369,
+		modtime: 1546544639,
+		compressed: `
+H4sIAAAAAAAC/+yZW3PTuBfA3/spzj90/gPbOE0DvZBh2elQmO0MbFlo94FOoIp9HB8qS0aSm5pOvvuO
+LOfi1E7cdhde9i2WdaRz10/OzQYAtDYpaPWhFRmT9Le3ZYLCl8IwEqh0R6rRNsVshNsxExSiNq22k9J+
+hDFbkPyqpfDcaC4WKBYar7u/7cYeFYKJkgkqQ6hbfcg1sKNMCGmYISkWh+2bALWvKLGv7GanEWko1sig
+0FQDU0MyiqkMYjQsYIZ14P101ruzj6eQagQTIcx3ApVy1B2nV7FbwoxBJd5XKVlM6dzstCe3hu0bkyVo
+ddRGkRi1Su8n86fJ4oZTGTn8ir6ZysymtHwpQhot+aRhyKauk6pk431DNwtVdhK2+nBesu+2N5J6H05D
+SyObUNVvF6JhNf2sI9bb3eufH3pvmBd2veeDm71nk81WhehkY+XAov9/kOa7O72S5ju9g/upPn8atOuL
+5BDmoYdQKlAYokLhkxgBgyGXw6W0X5HvtaX5Mwq0cZmuK9YGJVsXkXZt4ZeLuHL6itRZ9qR1hq+yxMiR
+YklEPvgR+pc6jcEtAjLMPea2bQOJ/LFwDly8YHwkFZkoftl/YXtEgMHLi3p35tl6zrzvNke3Hv/WP9/q
+fPEGs5Env/Tt70PvU9d7/usXb7C12WrfrQsuOSPGgNipE1jrj3WKH3qfCk3nP//3aPP/3ucvna3BTbe9
+09ubbDeb9lDDNH3HxtXy4fWfZ8cfXh/Ny0Yn6FNIqPOA2sXy6A4zg7o9Dbti47y+UJgOlMsOr0kbDVqC
+iZgBBj4nFAbGxDlE7MpWGeB1gr7BIF8/bxN5xrkVYYihVGiX9FFrEqMOHLt9OYqRiWZaoFGEVxjMJAOJ
+GoQ0EDPjR86Cwp6gEG6Xtvr4+8nZ2yP44+QUhghGpdpgUFX2MbumOI1bfXje6z19ut/rPt072H22v797
+0D2omE6imO41mj8NMAmDI1SrI5wq3qgfHgInV6pnH95qCJWMYRxR4ZipC2KWWdsDORZcsqDafDIY13a6
+UKqY2b7SShXdFm7Q7FY0NqYUy5oBjcJvKSkMbhGCq4naBlDdJmvOO0OG54qdvDqGV4UPj6pxZx1hcZah
+WkOdr5kfgXX/tMfmDnHn1hCBLZy3thRL7/NyszJDphHy3YAZIBHgNXQ78DEdavyWWgucKk7MRCgglJzL
+sd1UG+ZfglQBKnhMHey4THIS590BGDl94Cgeu59PvJ3BE6dQSIJxCImjzrQ1hLNMpsbtNS9UhTrlebay
+JOGZ5YW85J1iRuZ9I05MBgEp9I1UmVtfji18RpS0IZYBtoGJAKSJcmONomFqUE97BgkyxPjySsAUQipm
+zaLMKNXZ/zAWfjgO1xNxFVo2ossmgPkAOq5immac80PNaYzMje0pDQyW+83DCHq9J9Zx9E9C6bvQdAOg
+bsbUdSFbeWpVwXX1qbUy234aZf/ToH2/E30lcTeC7h/N3fezsw7A/2PwhzH43TH8HiS+FsarY16H5P8q
+la8D82Zs3vBbxEpPVUD6um9f9bReA+yrmL0S2yuO2jvA+8r+X7p2xCSOiyjsVNF/yTlz+HdE9xcq7RJj
+3ZfntW0iXw+u3IIdeJNXNOnpCMhwoRb9/GxuuxnTy8RF78KCNgqdKntp8C/HTNnKjhNmaEicTAZjstXP
+7W2gWDmH6yPpX2JxBbliPEXXIUhDSMgD13tsW/AjJkZYBux5WfeWPFs1XFOfebo5/1Yl15LDlz9zL1/K
+NmbpU86aY8v08K78l8Ryokw2/g4AAP//gVlZF+EYAAA=
+`,
+	},
+
+	"/tag-list.schema.json": {
+		name:    "tag-list.schema.json",
+		local:   "tag-list.schema.json",
+		size:    432,
+		modtime: 1546544639,
+		compressed: `
+H4sIAAAAAAAC/3SQwWrrQAxF9/4KMe8tk0x3BW+7KhS6yQ+otmIr2KNB0saU/HsZe5KaQJc+vudeNN8N
+AIT/3IcWwuie2xglU+okOXIitZPoEK0bacboOFic2DwcNm/jO/dqko4bXcVe8eLHl9fa8K+KPVmnnJ0l
+Ffk8skFWyaS+QN02QChbIBfAaYIyDpwAQSmLsYsup9pXXSYLLaw3FZpwpt13Qb7kgoK5chpC/XM73JUy
+8qSw0/zE/m7al+1TqIrLY655xIKzT2vi8+0dzuXEj9/3vdvydaXOQ3NrfgIAAP//IdRifbABAAA=
+`,
+	},
+
+	"/": {
+		name:  "/",
+		local: `.`,
+		isDir: true,
+	},
+
+	"/.git": {
```

Comment:
We really need that?

Reply:
This comes from another package but thanks for pointing that out. I'll clean up this up in the other repo.